basics/src/main/kotlin/de/thermondo/basics/Basics07.kt

```diff
@@ -0,0 +1,9 @@
+package de.thermondo.basics
+
+/**
+ * Enumeration is a named list of constants.
+ * In Kotlin, an enum has its own specialized type, indicating that something has a number of possible values.
+ * and it's defined by using enum keyword before a class.
+ *
+ * Define an enum class named Basics6Solution that prints the number of days of the week to the console.
```

Comment:
```suggestion
 * Define an enum class named Basics07Solution that prints the number of days of the week to the console.
```

Also, does the _enum_ class _itself_ print to console, or should it be a different class that prints to console?

Reply:
We don't need to print to the console at all. The unit test should validate that the enum class was created (it compiles) and that the values exist.